Move kthvalue scalar test to separate method for XLA

heitorschueroff opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#48042 Move kthvalue scalar test to separate method for XLA**

Moving scalar test to a separate method so the XLA team can continue to test for the other cases without failing. Requested here https://github.com/pytorch/xla/issues/2620#issuecomment-725696108

Differential Revision: [D25055677](https://our.internmc.facebook.com/intern/diff/D25055677)

heitorschueroff committed:
Move kthvalue scalar test to separate method for XLA

[ghstack-poisoned]

JackCaoG reviewed (approved):
Thanks @heitorschueroff !

mruberry commented:
This test is causing the XLA build to fail, though - does it need a decorator to be skipped on XLA for now?

heitorschueroff commented:
> This test is causing the XLA build to fail, though - does it need a decorator to be skipped on XLA for now?

Yes, I think adding the decorator is a good idea since I believe this failure with scalar input is expected by XLA. @JackCaoG, can you confirm this is correct?

heitorschueroff committed:
Update on "Move kthvalue scalar test to separate method for XLA"


Moving scalar test to a separate method so the XLA team can continue to test for the other cases without failing. Requested here https://github.com/pytorch/xla/issues/2620#issuecomment-725696108

[ghstack-poisoned]
